packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -86,4 +87,41 @@ test.describe('Participant pane tests', async () => {
 
     expect(await stableScreenshot(page)).toMatchSnapshot(`PSTN-call-screen-with-dialpad.png`);
   });
+
+  test.only('callee participant is displayed with connection state', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
```

Comment:
Any chance we can run these tests for both flavors (on stable we don't expect the connection string to be shown).

This is something I learnt from https://github.com/Azure/communication-ui-library/pull/2191

When writing beta-only features, we should think about browser tests that validate that the feature is *not* in the stable build.

Reply:
Good idea.

Reply:
Unfortunately, we can't do this for these tests as we are introducing a new participant pane all together and the same tests won't work for stable flavour

Reply:
Realized later we should write localization tests for this instead